great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -734,8 +736,9 @@ def _build_datetime_columns_rule() -> Rule:
             "estimator": "bootstrap",
             "n_resamples": 9999,
             "random_seed": None,
-            "quantile_statistic_interpolation_method": "auto",
-            "quantile_bias_std_error_ratio_threshold": 0.25,
+            "quantile_statistic_interpolation_method": "nearest",
```

Comment:
@NathanFarmer While you are at it, please change `round_decimals` from wherever it says `12` to be `15` in all variables dictionaries for both data assistant classes.  Thanks!

Reply:
✅